assets/js/common/PatchList/PatchList.stories.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+
+import { relevantPatchFactory } from '@lib/test-utils/factories/relevantPatches';
+import PatchList from './PatchList';
+
+const fakePatchArgs = {
+  patches: relevantPatchFactory.buildList(5),
+};
+
+export default {
+  title: 'Components/PatchList',
+  components: PatchList,
+  argTypes: {
+    patches: {
+      control: {
+        type: 'array',
+      },
+      description: 'A list of patches',
+    },
+  },
+  decorators: [
+    (Story) => (
+      <MemoryRouter>
+        <Story />
+      </MemoryRouter>
+    ),
+  ],
+  render: (args) => <PatchList {...args} />,
+};
+
+export const Default = {
+  args: fakePatchArgs,
```

Comment:
Could we avoid the intermediate variable?

It's better to have the structure right here to see, like { patches: yucallthefactory}

Reply:
yes, was an orphan from eralier code :D